tools/dbgen/generator.class.php

```diff
@@ -588,7 +588,7 @@ protected function _generateUsers($count) {
 
         // Ima admin
         $password = '$2y$10$StMpCySaSLHuvh3KLFNiZ.vSEykBF8kVWEG6ZtuXzQd4F9WWlfBFi';
-        echo "INSERT INTO `user` (`username`, `password`, `email`, `last_login`, `ID`, `verified`, `admin`, `full_name`, `active`, `twitter_username`, `request_code`) VALUES\n";
+        echo "INSERT INTO `user` (`username`, `password`, `verified`, `email`, `last_login`, `ID`, `verified`, `admin`, `full_name`, `active`, `twitter_username`, `request_code`) VALUES\n";
```

Comment:
Yep you're right, it was only the seed.sql file which was causing me problems.  It was unfair of me to assume it would be consistently broken :-/  I'll change this PR to only be for the seed.sql file.